spring-grpc-core/src/main/java/org/springframework/grpc/server/DefaultGrpcServerFactory.java

```diff
@@ -56,6 +51,9 @@ public class DefaultGrpcServerFactory<T extends ServerBuilder<T>> implements Grp
 
 	private final List<ServerBuilderCustomizer<T>> serverBuilderCustomizers;
 
+	@Autowired
```

Comment:
Since we do not yet have support for interceptors, lets not add that in this PR as I am actively working on #4. Instead you can use a `ServerBuilderCustomizer` as follows:

```java
@Bean
<T extends ServerBuilder<T>> ServerBuilderCustomizer<T> metricsInterceptor() {
	return (serverBuilder) -> serverBuilder.intercept(new MyXyInterceptor()));
}
```

If you add the above customizer in the auto-configuration then it will get applied at server build time.